src/NServiceBus.Testing.Fakes/Fakes/OutgoingMessage.cs

```diff
@@ -0,0 +1,31 @@
+namespace NServiceBus.Testing
```

Comment:
Talked to @timbussmann about this we propose to call YAGNI on the folder and just put the files in the root for now. We can add new projects or folders later if needed